src/main/webapp/app/admin/cleanup-service/cleanup-service.component.html

```diff
@@ -0,0 +1,76 @@
+<div>
+    <h2>
+        <span id="cleanup-page-heading" jhiTranslate="cleanupService.title">Cleanup Service Operations</span>
+    </h2>
+    <div class="table-responsive">
+        <table id="cleanupOperations" class="table table-striped" aria-describedby="cleanup-page-heading">
+            <thead>
+                <tr>
+                    <th scope="col" jhiTranslate="cleanupService.table.operation">Operation</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteFrom">Delete From</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteTo">Delete To</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.lastExecuted">Last Executed</th>
```

Comment:
I think we agreed to leave out the text when there's a translation key
```suggestion
                    <th scope="col" jhiTranslate="cleanupService.table.operation"></th>
                    <th scope="col" jhiTranslate="cleanupService.table.deleteFrom"></th>
                    <th scope="col" jhiTranslate="cleanupService.table.deleteTo"></th>
                    <th scope="col" jhiTranslate="cleanupService.table.lastExecuted"></th>
```